ownCloud/Theming/NSObject+ThemeApplication.swift

```diff
@@ -69,7 +69,7 @@ extension NSObject {
 
 				case .bigTitle:
 					themeButton.themeColorCollection = collection.neutralColors
-					themeButton.titleLabel?.font = UIFont.systemFont(ofSize: 22)
+					themeButton.titleLabel?.font = UIFont.systemFont(ofSize: 34)
```

Comment:
@hosy Would not use magic value as font size (at least a constant would be nice). And also have seen in another PR, that we started using dynamic type. Would it be an option here?

Reply:
Should be addressed in a new issue.